src/Model/ModelData.php

```diff
@@ -252,8 +252,7 @@ private function isAccessibleProperty(string $property): bool
     // -----------------------------------------------------------------------------------------------------------------
 
     /**
-     * Add methods from the {@link ModelData::$failover} object, as well as wrapping any methods prefixed with an
-     * underscore into a {@link ModelData::cachedCall()}.
+     * Add methods from the {@link ModelData::$failover} object
```

Comment:
I don't know if the underscore thing has _ever_ been true, but it definitely isn't right now.